test/ng/directive/selectSpec.js

```diff
@@ -1888,7 +1888,14 @@ describe('select', function() {
             optionElements = element.find('option');
             expect(optionElements.length).toEqual(1);
             expect(scope.obj.value).toEqual([]);
-            expect(element.val()).toBe(null);
+
+            // Cover both jQuery 3.x ([]) and 2.x (null) behavior.
+            var val = element.val();
```

Comment:
We have custom `toBeOneOf()` matcher. If we only had `toEqualOneOf()` :stuck_out_tongue: